compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -34,3 +165,81 @@ TEST_F(ResolveCustomOpAddPassTest, name)
   auto const name = _pass.name();
   ASSERT_NE(nullptr, name);
 }
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_CustomBroadcastTo)
+{
+  _g.init(true);
+
+  auto ret = _pass.run(_g.g());
+  EXPECT_EQ(true, ret);
+
+  auto add = dynamic_cast<luci::CircleAdd *>(_g.output(0)->from());
+  EXPECT_NE(nullptr, add);
+
+  // check if inputs are set after changing from Custom(AddV2) to Add.
+  // if the pass runs, x and y of Custom(AddV2) will be connected to x and y of Add.
+  auto input_x = dynamic_cast<luci::CircleInput *>(add->x());
+  EXPECT_NE(nullptr, input_x);
+
+  EXPECT_EQ(2, input_x->dim(0));
+  EXPECT_EQ(3, input_x->dim(1));
+
+  auto input_y = dynamic_cast<const luci::CircleConst *>(add->y());
+  EXPECT_NE(nullptr, input_y);
+
+  EXPECT_EQ(1, input_y->dim(0));
+  EXPECT_EQ(3, input_y->dim(1));
```

Comment:
@mhs4670go , as I know, BroadcastTo output shape should be changed to by doing broadcast to the shape input, but skipped doing so cause `Add` Op will do the broadcasting. I think we need to add some comments about this in the main code and here. What's your comment?

Reply:
You're right. That makes it easy to know about the pass.

Reply:
@mhs4670go 
> I think we need to add some NOTE comments about this in the main code and here. 
> You're right. That makes it easy to know about the pass.

I will add some NOTE comments to understand this. Thank you!